src/server/api/gutterRouter.ts

```diff
@@ -62,7 +62,7 @@ export const buildGutterRouter = (
                 abTestName: test.name,
                 abTestVariant: variant.name,
                 campaignCode: buildGutterCampaignCode(test, variant),
-                componentType: 'ACQUISITIONS_OTHER', // TODO: TBC - ACQUISITIONS_GUTTER? Changes will need to be made to the Ophan pipeline.
+                componentType: 'ACQUISITIONS_GUTTER',
```

Comment:
not a big deal, but I'm just wondering if it's worth separating this functional change out into its own follow-up PR. I think the rest of the changes in this branch should not affect the behaviour of client or server

Reply:
Sensible - reverting that bit.